### ISSUE TITLE
How to handle service configuration

### ISSUE THREAD

**Issue by TimMikeladze** [graphql-accounts/graphql-accounts#4]
In Meteor's accounts system the configuration for each service (keys, login style, etc) is stored in the database. @gadicc I see similar approach in your DBDriver interface with the `fetchConfig` function. 

Using Grant the services are configured through JS/JSON and the data probably originates from environment variables. 

My question is which implementation to go with? Or perhaps a hybrid of the two? 

What are the benefits of storing the service config in the database?

**Comment by gadicc** [comment]
Hybrid would be nice.  From a design perspective, as long as the Grant init is done async (or, that we can mutate the config post-init), it's pretty easy to offer both.

The Meteor guided config is great for less experienced devs and I think overall just more convenient, but I don't think essential.  Depending on how far we take it, we could (in the long term) allow to add new services all online without redeploying the app, or other runtime config options.  Would also be cool to leverage this data in existing Meteor installs, but we can always do that in a separate compatibility later.

In short, I think let's start off with JSON but try design the system in a way that accommodates DB config further down the line.

**Comment by TimMikeladze** [comment]
Sounds like a plan.
